src/Tribe/Attendee_Registration/Template.php

```diff
@@ -236,54 +221,6 @@ public function add_body_classes( $classes ) {
 		return $classes;
 	}
 
-	/**
-	 * Add the theme to the body class, in order to
-	 * add compatibility for official themes.
-	 *
-	 * @since 4.9
-	 * @param array $classes List of classes to filter
-	 *
-	 * @return array $classes
-	 */
-	public function theme_body_class( $classes ) {
```

Comment:
Lets deprecate the method since it's public. Return $classes and don't modify. We can remove the add_filter() usage but keep the method for now.